services/services.rst

```diff
@@ -0,0 +1,209 @@
+
+* `SuMIRe/PFS PBworks wiki`_
+* `Public Web Site at pfs.ipmu.jp`_
+* `PFS project internal website (pfs.ipmu.jp)`_
+* `Online meeting`_
+* `Services for software development at pfspipe.ipmu.jp and external`_
+
+SuMIRe/PFS PBworks wiki
+------
+
+"SuMIRe/PFS PBworks" is the only official wiki system for PFS. 
+
+======
+Login and ToC
+======
+
+After logging in at `PBworks <http://sumire.pbworks.com/>`_, 
+you will get a project index page named 
+"Subaru Measurement of Images and Redshifts (SuMIRe)". 
+This page contains links to contents in the wiki, such as 
+
+* PFS Project Office (General links, including link to telecon indexes)
+* Documents (Link list to documents)
+* Meeting, conference, etc.  (Link list to meetings)
+* Mailing lists (List of avail lists)
+* PFS working groups (Member list)
+
+======
+Page indexes
+======
+
+For some continuous meetings, index for each agenda/memo pages are avail.
+
+* `Systems Engineering group telecon <http://sumire.pbworks.com/Systems-engineering-telecons>`_
+* `Manager group telecon <http://sumire.pbworks.com/Management-telecons>`_
+
+
+======
+Files
+======
+
+Also, you can find all files uploaded into this SuMIRe/PFS PBworks from 
+`folder listing <http://sumire.pbworks.com/w/browse/#view=ViewAllFiles>`_.
+Some files are categorized into *FOLDERS*, and you can get each list by 
+clicking FOLDER name at left side. 
+
+======
+Editing manual
+======
+
+You can find `manual for editing PBworks <http://usermanual.pbworks.com/>`_. 
+
+
+Public Web Site at pfs.ipmu.jp
+------
+
+Public project information is at http://pfs.ipmu.jp , 
+such as list of meetings, list of publications, and instrument parameters. 
+
+PFS project internal website (pfs.ipmu.jp)
+------
+
+Every contents at https://pfs.ipmu.jp/ are project only, and you will 
+be required to log in to view pages. 
+
+If you have any issue on this site, contact administrator 
+at pfs at pfs.ipmu.jp . 
+
+======
+Login and user account
+======
+
+Use your 'account name' (not email address) and 'password'.
+For your first time, please follow notification email to change your password 
+from an initial one (randomly created). 
+
+You can edit your account information from `LDAP account manipulator service <https://pfs.ipmu.jp/ldap-manip/>`_, 
+such as password, your real name, institution, and photo. 
+Also you can view list of all accounts from 
+
+* `List of existing accounts <https://pfs.ipmu.jp/ldap-manip/view_all.cgi>`_
+* `photo directory <https://pfs.ipmu.jp/ldap-manip/view_allphoto.cgi>`_
+
+======
+ToC
+======
+
+When accessing to https://pfs.ipmu.jp/, you will get newest list of 
+contents in this server. 
+
+LDAP account manipulator
+  You can view your account setting, list of all avail accounts, and photo 
+  directory. 
+  Also, you can edit your account setting (real name, password, institution, 
+  photo, etc.) from this service.
+Content sharing services
+  Available services:
+
+  * Document server for PFS
+  * Photo archive
```

Comment:
Link to photo archive, as you do for WebDAV?

Reply:
let me left for 3rd round